utils/README.md

```diff
@@ -63,3 +66,87 @@ To install and configure a system with Kata Containers and containerd, run:
 ```bash
 $ bash -c "$(curl -fsSL https://raw.githubusercontent.com/kata-containers/kata-containers/main/utils/kata-manager.sh)"
 ```
+
+### Choose a Hypervisor
+
+Kata works with different [hypervisors](../docs/hypervisors.md). When you install a Kata system, the default hypervisor
+will be configured, but all the available hypervisors are installed.
+This means you can switch between hypervisors whenever you wish.
+
+#### List available hypervisors
+
+Run the following on an installed system:
+
+```bash
+$ bash -c "$(curl -fsSL https://raw.githubusercontent.com/kata-containers/kata-containers/main/utils/kata-manager.sh) -L"
+```
+
+#### Show the default packaged hypervisor
+
+To show the default packaged hypervisor, run the following on an
+installed system:
+
+```bash
+$ bash -c "$(curl -fsSL https://raw.githubusercontent.com/kata-containers/kata-containers/main/utils/kata-manager.sh) -L" | grep default
+```
+
+#### Show the locally configured hypervisor
+
+`kata-manager.sh` will create a "local" copy of the packaged Kata configuration
+file.
+
+To show details of the _local_ copy of the configuration files on an
+installed system, run:
+
+```bash
+$ bash -c "$(curl -fsSL https://raw.githubusercontent.com/kata-containers/kata-containers/main/utils/kata-manager.sh) -e"
+```
+
+> **Note:** This command can only be run once Kata has been installed.
+
+> See the [configuration documentation](https://github.com/kata-containers/kata-containers#configuration)
+> for further information.
+
+#### Switch hypervisor
+
+> **Note:**
+>
+> If you create your own local configuration files, you should ensure
+> they are backed up safely before switching hypervisor configuration
+> since the script will overwrite any files that it needs to create in
+> the Kata configuration directory (`/etc/kata-containers/` and
+> sub-directories).
+
+##### To install Kata Containers and containerd and configure it for a specific hypervisor
+
+To specify that Kata be installed and configured to use a specific
+hypervisor, use the `-H` option. For example, to select Cloud Hypervisor, run:
+
+```bash
+$ bash -c "$(curl -fsSL https://raw.githubusercontent.com/kata-containers/kata-containers/main/utils/kata-manager.sh) -H clh"
+```
+
+> **Note:** See the [List available hypervisors](#list-available-hypervisors) section
```

Comment:
This note is not that much useful as we don't have a way to actually list the hypervisors before we install Kata Containers. :-/

Reply:
The doc now lists the "short names".

However, I am wondering about removing the `-H` option since, effectively, that option just installs Kata, then calls `-S`. There isn't much benefit to providing `-H` since we still need to install Kata. Then the `-S` just does the config switch. wdyt to removing `-H`?